source/_integrations/litterrobot.markdown

```diff
@@ -0,0 +1,111 @@
+---
+title: Litter-Robot
+description: Instructions on how to integrate a Litter-Robot WiFi-enabled, automatic, self-cleaning litter box to Home Assistant.
+ha_category:
+  - Vacuum
+ha_iot_class: Cloud Polling
+ha_release: 2021.3
+ha_config_flow: true
+ha_quality_scale: gold
+ha_codeowners:
+  - '@natekspencer'
+ha_domain: litterrobot
+---
+
+The Litter-Robot integration allows you to control and monitor your WiFi-enabled, automatic, self-cleaning litter box for cats.
```

Comment:
We use the `Wi-Fi` spelling.

<https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>